pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -200,17 +256,6 @@ func TestGetPipelineFunc(t *testing.T) {
 			Name: "simple",
 		},
 		expected: simplePipelineWithBaseSpec(),
-	}, {
-		name:           "remote-pipeline-without-defaults",
```

Comment:
I still think it would be better to move this to TestGetPipelineFunc_RemoteResolution rather than removing it entirely